shake-bench/src/Development/Benchmark/Rules.hs

```diff
@@ -378,6 +381,11 @@ svgRules build = do
         title = show (unescapeExperiment exp) <> " - live bytes over time"
     plotDiagram False diagram out
 
+eventlogRules :: FilePattern -> Rules ()
+eventlogRules build = do
+  build -/- "*/*/*.eventlog.html" %> \out -> do
+    need [dropExtension out]
+    cmd_ $ "eventlog2html " <> dropExtension out
```

Comment:
Why not `cmd_ "eventlog2html" [dropExtension out]`? I think that would be more space-safe and avoids us joining strings by hand.